content/zh/news/releases/1.16.x/announcing-1.16.1/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: 发布 Istio 1.16.1
+linktitle: 1.16.1
+subtitle: 补丁发布
+description: Istio 1.16.1 补丁发布。
+publishdate: 2022-12-12
+release: 1.16.1
+---
+
+此版本包含了一些改进稳健性的漏洞修复。
+此发布说明描述了 Istio 1.16.0 和 Istio 1.16.1 之间的不同之处。
+
+此版本包括（2022-12-06 发布的）Go 1.19.4 中对 `os` 和 `net/http` 包的安全修复。
+
+{{< relnote >}}
+
+## 变更{#changes}
+
+- **弃用** 弃用了对 Kubernetes 1.20 之前的版本使用 `PILOT_CERT_PROVIDER=kubernetes`。
+
+- **更新** 将 Kiali 插件版本更新为 1.59.1。
+
+- **修复** 修复了 OpenTelemetry 跟踪器不工作的问题。
+  ([Issue #42080](https://github.com/istio/istio/issues/42080))
+
+- **修复** 修复了正使用的 Wasm 缓存文件被标记为陈旧且被删除的问题。 ([Issue #42154](https://github.com/istio/istio/issues/42154))
```

Comment:
can you update this in en part, it seems reverted by https://github.com/istio/istio/pull/42187

Reply:
Has removed this **Fixed** from en and zh. Is my change correct?